src/extension/features/budget/days-of-buffering/index.js

```diff
@@ -58,6 +58,15 @@ ${i10n('budget.dob.avgOutflow', 'Average daily outflow')}: ~${formatCurrency(ave
     }
   }
 
+  /**
+   * Days of buffering is calculated by taking the total outflows within the selected window
+   * and dividing it the number of days in the selected window. The selected window comes from
```

Comment:
"... and dividing it **by** the number ..." ?

Reply:
I re-read this four times before my brain realized "by" wasn't there. Fixed.